### ISSUE TITLE
Goes to "armed_home" at restart of Home Assistant

### ISSUE THREAD

**Issue by J-Lindvig** [nielsfaber/alarmo#85]
Hello there....

Firstly let me thank you for your great work on Alarmo, I am very impressed and fond of it.

Sadly I am facing some troubles and undesirable behaviour. When I restart or reboot my Home Assistant, the alarm is not returning to the previous state "disarmed", but to "armed_home". This makes the alarm trigger shortly after - leaving me a very unpopular husband and father.

Here is my log:
`2021-01-04 20:01:42 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] async_added_to_hass
2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] Initial state is disarmed
2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.automations] state is updated from None to disarmed
2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] alarm_arm_home
2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.automations] state is updated from disarmed to armed_home`

Could you please investigate in this matter or guide me in assisting you in further debugging?

Kind regards

Jacob

**Comment by nielsfaber** [comment]
Hello Jacob,

Thanks for liking Alarmo!
I'm planning to add more stuff over time, so hopefully you will like it more even.

Thanks for enabling the debug logging, it makes it much easier to help you.

This line actually tells you what state Alarmo starts in:
> 20:01:43 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] Initial state is disarmed 2021-01-04 20:01:43

Alarmo detected that before restarting HA, the alarm was in `disarmed`, so it will return to that state.
(For armed state it would also work, but first Alarmo waits for all sensors to be initalized).

What happens after is this:
>  2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.automations] state is updated from None to disarmed
2021-01-04 20:01:43 DEBUG (MainThread) [custom_components.alarmo.alarm_control_panel] alarm_arm_home 

So Alarmo updates its state to `disarmed` and not even a second later it receives the command `alarm_arm_home`.

The command `alarm_arm_home` is triggered by an external source.
Either:
* You have an automation that sends this command
* You use the MQTT interface and the command is sent here (or the command is a retained payload)

In other words, the problems are caused by your own configuration, I cannot help you investigate further.
Sadly I don't have any debug logging in place to be able to tell you it's coming from MQTT, but I will add this for the future.

**Comment by J-Lindvig** [comment]
Hi Niels

I will investigate in the MQTT retain matter - this could be the culprit, and I will return with more info.

/Jacob

**Comment by J-Lindvig** [comment]
Solved....

An old MQTT state with retain: true was the culprit.
I have an 433 MHz keypad, which originally did have the retain switch to true. When I discovered the issue, I changed the config to false.... But I didn't send a new state, therefore the old state was stille firing a "ARM_HOME".

Problem solved. Thanks Niels.
